addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -128,6 +132,7 @@ function ActivityFactory({ Entity }) {
                 res_model,
                 state,
                 summary,
+                _isRecordDeleted: false,
```

Comment:
This is strange, can we remove this variable if we apply the await of my other comment?